### DISCUSSION TITLE
Android Studio as main IDE

### DISCUSSION THREAD

**Discussion by Rai0** [korlibs/korge#2203] [Q&A]
Hello there! Im new into the engine and I want to use it for android game development. Im a bit confused as I cannot find android examples, especially on YouTube. Is it possible to use Android Studio as the main IDE? Can I use KorGE editor in it?

**Comment by soywiz** [comment]
Right now KorGE Plugin is limited to IntelliJ IDEA.

What's your use case? Do you want to add a game into an existing android application? Is there something preventing from using IntelliJ IDEA?

**Reply by Rai0** [reply]
It took a while huh. Im planning to write an app using some KorGE features on some screens, i havent started it yet. Do you have some documentation how to do such a thing? Or any simple advices how to start? :)

**Reply by soywiz** [reply]
You can use KorGE as library without the plugin. Here is an old example showing how to integrate it with Android:

https://github.com/korlibs/korge-hello-world-integrated-android
